### ISSUE TITLE
aria-controlled validation

### ISSUE THREAD

**Issue by BladeBronson** [ealush/emoji-picker-react#415]
The search field correctly specifies an `id` for its `aria-controlled` attribute, but the specified element is conditionally rendered. My company's accessibility tests are failing when using this package because of this.
